geomesa-accumulo/geomesa-accumulo-distributed-runtime/pom.xml

```diff
@@ -28,24 +28,6 @@
             <artifactId>geomesa-accumulo-datastore_2.11</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-accumulo-raster_2.11</artifactId>
```

Comment:
@aheyne have you pushed the changes? i would be wary of having 2 shaded jars with duplicate classes... ideally the raster jar would only include the things not already in the vector jar. that might be tricky to get and keep correct though.
have you tested out the accumulo and raster quickstarts with the changes?